ocp3config/ocp3_test.go

```diff
@@ -0,0 +1,55 @@
+package ocp3
+
+import (
+	"testing"
+
+	configv1 "github.com/openshift/api/legacyconfig/v1"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConfigParseMaster(t *testing.T) {
+	testConfig := Config{
+		Masterf: "../examples/source/etc/origin/master/master-config.yaml",
```

Comment:
I guess we want to have test data input along the code, somewhere in ocp3config directory.

Reply:
How do see it?